pypiserver/_app.py

```diff
@@ -389,12 +391,29 @@ def json_info(project):
     if not packages:
         raise HTTPError(404, f"package {project} not found")
 
+    package_links = defaultdict(list)
+    for pkg in packages:
+        package_links[pkg.version].append(pkg.relfn_unix)
+    # links = [pkg.relfn_unix for pkg in packages]
+
     latest_version = packages[0].version
     releases = {}
     req_url = request.url
-    for x in packages:
-        releases[x.version] = [
-            {"url": urljoin(req_url, "../../packages/" + x.relfn)}
+    for package in packages:
```

Comment:
Thanks for renaming this to `package`, imho, I like it also a bit better this way 👍